examples/README.md

```diff
@@ -35,3 +36,7 @@ Now open another terminal (ideally side by side to the old one) and run the clie
 ```bash
 ./submit_example_problems.py
 ```
+
+## Testing:
```

Comment:
I agree and I changed the structure accordingly. However, it's unfortunately not possible to have two packages called `tests` (the main and the example test suite) because pytest complains about it. I now renamed the stream calc test suite to sc_tests.